src/styles/global/global.scss

```diff
@@ -155,6 +157,7 @@ img {
 }
 
 .nav-tabs::-webkit-scrollbar {
```

Comment:
move this style statement into the nav-tabs block ( e.g. to line 141 ) using the & operator:
  &::-webkit-scrollbar {.....